src/layman/util.py

```diff
@@ -51,7 +51,13 @@ def to_safe_names(unsafe_names, type_name):
 def check_username_decorator(f):
     @wraps(f)
     def decorated_function(*args, **kwargs):
-        check_username(request.view_args['username'])
+        username = request.view_args['username']
+        if request.method == 'POST':
```

Comment:
Should there be tested, which POST method it is? I think, that only some POST methods can create user.

Reply:
We could limit it only to POST Layers and POST Maps, but I think it’s not necessary. Creating user workspace on Post Layer Chunk does not break anything.
Anyway, if it makes sense to you to limit POST requests&check_username_decorator, I suggest to create separate issue for this and leave it for later.

Reply:
I am not sure what happened if I call POST with non existing username. I will check it and then either let it go or create separate issue.